AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsInternal.h

```diff
@@ -14,10 +14,14 @@ NS_ASSUME_NONNULL_BEGIN
  * @param eventName  Event name.
  * @param properties The typed event properties.
  * @param transmissionTarget  The transmission target to associate to this event.
+ * @param flags      Optional flags. Events with MSFlagsPersistenceCritical will be considered as a higher priority than events with
+ * MSFlagsPersistenceNormal or MSFlagsDefault, will be removed at the latest when the storage is full and sent prior to lower priority
```

Comment:
> will be removed at the latest when

I feel like something is missing between latest and when, maybe some wording can be improved here, it looks odd to me.

Reply:
(Multiple place comment)

Reply:
Maybe something like this?

"Optional flags. Events with MSFlagsPersistenceCritical have a higher priority than those with MSFlagsPersistenceNormal or MSFlagsDefault. When device storage is full, lower priority events will be removed before higher priority ones."

Reply:
Maybe: `Events tracked with the MSFlagsPersistenceCritical flag will take precedence over all other events in storage. An event tracked with this option will only be dropped if storage must make room for a newer event that is also marked with the MSFlagsPersistenceCritical flag.`